src/citrine/informatics/predictors/predictor.py

```diff
@@ -1,125 +1,11 @@
-import warnings
-from typing import Optional, Type, List
-from uuid import UUID
-
 from citrine._rest.asynchronous_object import AsynchronousObject
-from citrine._serialization import properties
-from citrine._serialization.polymorphic_serializable import PolymorphicSerializable
-from citrine._session import Session
-from citrine.informatics.data_sources import DataSource
-from citrine.resources.report import ReportResource
-from citrine.informatics.predictors.single_predict_request import SinglePredictRequest
-from citrine.informatics.predictors.single_prediction import SinglePrediction
-from citrine._utils.functions import format_escaped_url
 
 __all__ = ['Predictor']
 
 
-class Predictor(PolymorphicSerializable['Predictor'], AsynchronousObject):
-    """Module that describes the ability to compute/predict properties of materials.
-
-    Abstract type that returns the proper type given a serialized dict. Subtype
-    based on the 'type' value of the passed in dict.
-
-    """
+class Predictor(AsynchronousObject):
```

Comment:
I added back this type as pretty much a dummy object, in case anyone is importing it within their scripts. The majority of fields have been moved over to Graph Predictor.